### PR TITLE
Add tqm to docs

### PR DESCRIPTION
**Pull request by RaneyDazed**
Added documentation on tqm to the docs repo. Also added core to recommended install types for @keldian in omegabrr.md

- [x] App documentation page created
- [x] Reference added in `docs/sandbox/index.md`
- [x] Reference added in `mkdocs.yml`

### CODE REVIEW

**Comment by RaneyDazed** [comment]
Ah I didn't add anything for editing the service file.

**Comment by owine** [comment]
Seems like the docs should have the user setup the config before running the role.

**Comment by RaneyDazed** [comment]
True. I should reorganize.

File: docs/sandbox/apps/tqm.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+| Details     |
+|-------------|
+| [:octicons-mark-github-16: Github](https://github.com/l3uddz/tqm){: .header-icons target=_blank rel="noopener noreferrer" }|
+
+Recommended install types: Feederbox, Saltbox, Core
+
+### 1. Setup
+
+Edit in your favorite code editor  (with json highlighting) or even a unix editor like nano.
```
Comment (owine) [inline docs/sandbox/apps/tqm.md:20]: JSON?

Comment (RaneyDazed) [reply docs/sandbox/apps/tqm.md:20]: Oh whoops. damnit. I stole it from cloudplow.md

Comment (RaneyDazed) [reply docs/sandbox/apps/tqm.md]: Fixed. going to add a bit about adjusting the service file. and changing the settings.yml file

**Comment by RaneyDazed** [comment]
I don't like the settings.yml bit I added. Should I edit that down and remove the example? just leave the plain text?

**Comment by owine** [comment]
Shouldn't be instructing editing to the service file directly as it will be overwritten on any run of the role.

**Comment by RaneyDazed** [comment]
> Shouldn't be instructing editing to the service file directly as it will be overwritten on any run of the role.

Heya owine, should I leave that service file example as an example or just remove completely?

**Comment by RaneyDazed** [comment]
I think we're good now want to take a look?

File: docs/sandbox/apps/tqm.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      📢 tqm will not create a category for you, so be sure to create the category first. If you
+      want the file moved as well, you will need to set **Default Torrent Management Mode: Automatic**.
+
+`name:` The category (label) you want the torrent changed to.
+
+`update:` Define what is to be moved by tqm.
+
+### Modify the "Settings" file
+
+If you don't feel comfortable editing the service file directly, you can edit the **settings.yml** file in `/opt/sandbox/`. Once you set your download client, run the role again and it will update the service.
```
Comment (owine) [inline docs/sandbox/apps/tqm.md:143]: Remove reference to service file

**Review by owine — CHANGES_REQUESTED** [review]

File: docs/sandbox/apps/tqm.md (hunk 2 above)
Comment (RaneyDazed) [reply docs/sandbox/apps/tqm.md:143]: Ok I see, just a sec

**Comment by RaneyDazed** [comment]
That should be better. Is it ok I left the service bit at the end? Just stating what it would change.

**Pull request merged**